paasta_tools/cli/utils.py

```diff
@@ -490,16 +490,21 @@ def run_paasta_serviceinit(subcommand, master, service, instancename, cluster, *
         delta = "--delta %s" % kwargs['delta']
     else:
         delta = ''
-    command = 'ssh -A -n %s sudo paasta_serviceinit %s%s%s %s %s' % (
+    if 'stream' in kwargs and kwargs['stream']:
+        stream = kwargs['stream']
+    else:
+        stream = False
+    command = 'ssh -A -n %s sudo paasta_serviceinit %s%s-s %s -i %s %s %s' % (
```

Comment:
eh, this is getting a bit difficult to read. How about using the dict formatting?

```
'ssh -A -n %(master) sudo paasta_serviceinit %(verbose_flag) %(app_id_flag) -s %(service) -i %(instancenames) %(subcommand) %(delta)' % {
    'master': master,
    'verbose_flag': verbose_flag,
    ...
}
```